app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com_d = command.split()
+    if com_d[0] != "mv":
+        return None
+    n_dir = com_d[2].split("/")
```

Comment:
Better to use `os.path.split()`
```suggestion
    directory, filename = os.path.split(com_d[2])
```